src/components/TrancheInvestment.tsx

```diff
@@ -93,8 +91,12 @@ export const TrancheInvestment: FunctionComponent<TrancheInvestmentProps> = ({
 
 	useEffect(() => {
 		if (userTrancheBalance && currentReturns && amountWithdrawn) {
-			const currentValue =
-				userTrancheBalance.uiAmount + currentReturns.toNumber() - amountWithdrawn.uiAmount;
+			const programAmountCurrentReturns = toProgramAmount(currentReturns);
```

Comment:
same here. there are also some unused imports again but I think we need to fix that at the linter level.